app/views/notification_mailer/_status.html.haml

```diff
@@ -28,10 +28,10 @@
                               - if status.spoiler_text?
                                 %div.auto-dir
                                   %p
-                                    = Formatter.instance.format_spoiler(status)
+                                    = status.spoiler_text
 
                               %div.auto-dir
-                                = Formatter.instance.format(status)
+                                = status_content_format(status)
```

Comment:
Lacking custom emoji rendering.

Reply:
The original line lacks `custom_emojify: true`, so we never rendered emojis in the e-mails.

Reply:
Hmmm… maybe we could? Anyway since it doesn't change the behavior, it's fine by me.